docs/pages/tutorial.rst

```diff
@@ -46,7 +57,20 @@ Add the view to your ``urls.py``::
         url(r'^admin/', admin.site.urls),
         url(r'^people/', people)
     ]
+    
+If you are using Django 2.0 and above::
```

Comment:
I think I prefer using one django version for the tutorial. So we should assume 2.0 and rewrite everything to suit that.
You can also remove the function based views and just use a CBV. Please keep it simple though and use a `SingleTableView`